modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbStorageEngine.java

```diff
@@ -177,34 +198,23 @@ public RocksDbTableStorage createMvTable(
             StorageTableDescriptor tableDescriptor,
             StorageIndexDescriptorSupplier indexDescriptorSupplier
     ) throws StorageException {
-        RocksDbDataRegion dataRegion = regions.get(tableDescriptor.getDataRegion());
+        String regionName = tableDescriptor.getDataRegion();
 
-        int tableId = tableDescriptor.getId();
+        RocksDbStorage storage = storageByRegionName.get(regionName);
 
-        assert dataRegion != null : "tableId=" + tableId + ", dataRegion=" + tableDescriptor.getDataRegion();
+        assert storage != null : "No instances exist for region " + regionName;
```

Comment:
I would add information about the tableId to the error message.

Reply:
Fixed